### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by JohnN193**
figured I would update the readme before releasing the new module. Will put out a new release and update the canarys after merging this

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,13 @@
 This module implements the [`rdk:component:board` API](https://docs.viam.com/components/board/#api) and [`rdk:component:servo` API](https://docs.viam.com/components/servo/#api)
 
 Two models are provided in this module:
```
Comment (oliviamiller) [inline README.md:5]: remove the two

File: README.md [hunk 2]
```diff
@@ -59,7 +65,7 @@ Fill in the attributes as applicable to your board:
 
 ### Attributes
 
-The following attributes are available for `viam:raspberry-pi:rpi` board:
+The following attributes are available for `viam:raspberry-pi:<raspberry-pi-model>` board:
```
Comment (oliviamiller) [inline README.md:68]: can't comment on the exact line but should the pull up/down attributes be added to config attributes above?

Comment (JohnN193) [reply README.md:68]: pulls are described in the pins section, though I agree that the order/structure of the readme is kinda hard to follow

Comment (JohnN193) [reply README.md:68]: can be explicit in saying pins let you set the pull as well

File: README.md [hunk 3]
```diff
@@ -2,8 +2,14 @@
...
-* `viam:raspberry-pi:rpi` - Configure a Raspberry Pi 4, 3 and Zero 2 W,  board to access GPIO functionality: input, output, PWM, power, serial interfaces, etc.
+The following models are provided in this module:
+* `viam:raspberry-pi:rpi5` - Configure a Raspberry Pi 5 board to access GPIO functionality: input, output, PWM, power, serial interfaces, etc.
+* `viam:raspberry-pi:rpi4` - Configure a Raspberry Pi 4 board
+* `viam:raspberry-pi:rpi3` - Configure a Raspberry Pi 3 board
+* `viam:raspberry-pi:rpi2` - Configure a Raspberry Pi 2 board
+* `viam:raspberry-pi:rpi1` - Configure a Raspberry Pi 1 board
+* `viam:raspberry-pi:rpi0` - Configure a Raspberry Pi 0 board
+* `viam:raspberry-pi:rpi0_2` - Configure a Raspberry Pi 0 2 W board
 * `viam:raspberry-pi:rpi-servo` - Configure a servo controlled by the GPIO pins on the board.
```
Comment (oliviamiller) [inline README.md:13]: Will the rpi-servo work with pi5? I think the answer is no so we should specify this in readme

File: README.md (hunk 2 above)
Comment (oliviamiller) [reply README.md:68]: ah okay I see now, thanks for clarifying

File: README.md (hunk 3 above)
Comment (JohnN193) [reply README.md:13]: i believe it will work? rand and I had actually talked about removing the rpiservo logic to just use the gpio servo logic in rdk, but we never got around to it

Comment (oliviamiller) [reply README.md:13]: oh okay I assumed it didn't because it uses pigpio to do PWM which doesn't support the pi5? But i am in support of removing it

**Review by oliviamiller — APPROVED** [review]
